lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -960,9 +960,9 @@
     <datatype extension="storm.check" type="galaxy.datatypes.text:StormCheck" display_in_upload="true"/>
     <datatype extension="ctl.result" type="galaxy.datatypes.text:CTLresult" display_in_upload="true"/>
     <!-- CASTEP types -->
-    <datatype extension="castep" type="galaxy.datatypes.text:Castep" display_in_upload="true"/>
-    <datatype extension="param" type="galaxy.datatypes.text:Param" display_in_upload="true"/>
-    <datatype extension="den_fmt" type="galaxy.datatypes.text:FormattedDensity" display_in_upload="true"/>
+    <datatype extension="castep" type="galaxy.datatypes.text:Castep" display_in_upload="true" description="numerical output of a CASTEP job in ASCII format"/>
+    <datatype extension="param" type="galaxy.datatypes.text:Param" display_in_upload="true" description="the param file contains all the input control parameters required to run a CASTEP job. Consists of lines containing keywords and associated values"/>
+    <datatype extension="den_fmt" type="galaxy.datatypes.text:FormattedDensity" display_in_upload="true" description="formatted electronic density file output by CASTEP containing value of electron density for a grid of points"/>
```

Comment:
Couldn't find anything on the CASTEP docs for this, which is a shame - don't know why they don't have anything for it but even just the description is good.

Reply:
Ok I shall leave the CASTEP doc as is